Library/Formula/percona-toolkit.rb

```diff
@@ -1,3 +1,10 @@
+class AnyMySQLRequirement < Requirement
+  fatal true
+  default_formula "mysql"
+
+  satisfy { which("mysql_config") }
+end
```

Comment:
Ah, misinterpreted the `default_formula` part, thought it would install that formula only if `satisfy` would fail.